control-plane/pkg/reconciler/consumergroup/consumergroup.go

```diff
@@ -105,6 +110,48 @@ func (r Reconciler) ReconcileKind(ctx context.Context, cg *kafkainternals.Consum
 	return nil
 }
 
+func (r *Reconciler) FinalizeKind(ctx context.Context, cg *kafkainternals.ConsumerGroup) reconciler.Event {
```

Comment:
With this new `FinalizeKind` method, I was expecting to see that tests in `consumergroup_test.go` for `ReconcileKind` (TestReconcileKind) would have a new finalizer update event emitted.

Reply:
Thank you!!!